pages/[user-id]/my-profile.js

```diff
@@ -21,45 +30,70 @@ export default function MyProfile({ userInfo }) {
     photo: userInfo.photo
   })
 
-  // refactor later
-  // useEffect(() => {
-  //   fetchUserInfo();
-  // }, [])
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
 
-  // const fetchUserInfo = () => {
-  //   getUserInfo(userID).then(res => {
-  //     setCurrentUserProfileInfo({
-  //       fullName: res.name.first_name + ' ' + res.name.last_name,
-  //       location: res.location,
-  //       description: res.description,
-  //       endorsements: res.endorsements,
-  //       photo: res.photo
-  //     })
+  const handleClose = () => {
```

Comment:
Clearer naming. What is open? What is close? Can make a trello for later tech debt/refactoring